### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lachlansneff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 ## What is Nebulet?
 
-Nebulet is a microkernel that executes WebAssembly modules instead of ELF binaries. Furthermore, it does so in ring 0 and in the same address space as the kernel, instead of in ring 3. Normally, this would be super dangerous, but WebAssembly is designed to run safely on remote computers, so it can be securely sandboxed without losing performance.
+Nebulet is a microkernel that executes WebAssembly modules in ring 0 and a single address space to increase performance. This allows for low context-switch overhead, syscalls just being function calls, and exotic optimizations that simply would not be possible on conventional operating systems. The WebAssembly is verified and, due to a trick used to optimize out bounds-checking, unable to even represent the act of writing or reading outside its assigned linear memory.
```
Comment (gmorenz) [inline README.md]: `verified and, due` should be `verified, and due`

**Comment by lachlansneff** [comment]
This will fail building due to an issue with a dependency of nebulet and cretonne.

**Review by gmorenz — CHANGES_REQUESTED** [review]
Other than the grammar nitpick, lgtm.

File: README.md (hunk 1 above)
Comment (lachlansneff) [reply README.md]: Thanks for catching that!

**Review by gmorenz — APPROVED** [review]

**Pull request merged**